lab-lee/README.md

```diff
@@ -0,0 +1,37 @@
+##Lab-06 TCP Chat Creation
+
+**About This Project**
+
+This project's purpose is to create a locally hosted chat server that allows several local instances to connect and communicate with one another.
+
+`client.js` creates a user object for the chatroom every time someone connects to the room.
+
+`server.js` handles the logic for creating the server, handling user input, and appropriately displaying the correct messages to other users.
+
+Once connected to the chat, you may run `/help` for a list of available chat commands.
+
+**How to Run this Project**
+
+Check out the [package.json] (https://github.com/broxsonl/lab-06-tcp-chat/blob/master/lab-lee/package.json) file for the project to see the specific Developer Dependencies required to run this project. Or, enter the following command in your local copy of the repository:
+
+        npm  i -D eslint gulp-eslint gulp-mocha mocha mocha-eslint
+
+**How to connect to the server**
+
+In your terminal, navigate to the top of this project's directory. Run the following command:
```

Comment:
I understand what you mean, but the correct terminology would be "root" of the project for future reference :)